test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetListCommandTest.cs

```diff
@@ -850,7 +849,7 @@ public void ListCommand_InvalidInput_V2_NonExistent(string invalidInput)
                 result.Item1 != 0,
                 "The run did not fail as desired. Simply got this output:" + result.Item2);
 
-                Assert.Contains($"Unable to load the service index for source {invalidInput}.", result.Item3);
+                Assert.Contains($"Unable to load the service index for source", result.Item3);
```

Comment:
I'm trying to move us away from Item1/Item2/Item3. Please use the better named `AllOutput` here

Reply:
fixed in next iteration.